batteries.opam

```diff
@@ -19,7 +19,8 @@ build: [
 install: [make "install"]
 run-test: [make "test"]
 depends: [
-  "ocaml" {>= "4.00.0" & < "4.15.0"}
+  "ocaml" {>= "4.00.0"}
+  ("ocaml" {< "5.0"} | "camlp-streams")
```

Comment:
```suggestion
  "camlp-streams"
```

Reply:
we should bump the ocaml lower-bound then

Reply:
in addition to depending on camlp-streams